src/game/level.c

```diff
@@ -89,10 +99,50 @@ static bool Level_LoadFromFile(
         return false;
     }
 
+    if (!Level_LoadMeshBase(fp)) {
+        return false;
+    }
+
+    if (!Level_LoadMeshes(fp)) {
+        return false;
+    }
+
+    if (!Level_LoadAnims(fp)) {
+        return false;
+    }
+
+    if (!Level_LoadAnimChanges(fp)) {
+        return false;
+    }
+
+    if (!Level_LoadAnimRanges(fp)) {
+        return false;
+    }
+
+    if (!Level_LoadAnimCommands(fp)) {
+        return false;
+    }
+
+    if (!Level_LoadAnimBones(fp)) {
+        return false;
+    }
+
+    if (!Level_LoadAnimFrames(fp)) {
+        return false;
+    }
+
     if (!Level_LoadObjects(fp)) {
         return false;
     }
 
+    if (!Level_LoadStaticObjects(fp)) {
+        return false;
+    }
+
+    if (!Level_LoadTextures(fp)) {
+        return false;
+    }
```

Comment:
Actually we don't need to make these return a bool, as any error during level load is fatal and should cause the game to exit immediately.